source/install-guide/installing-from-zip.rst

```diff
@@ -0,0 +1,21 @@
+.. _installing-from-zip:
+
+Installing using a ZIP file (Linux/macOS)
```

Comment:
You can remove `Linux/macOS`, this works for everyone.

Reply:
Yes, that's true, especially if I remove the Windows installer section